<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PreferenceDialog</class>
 <widget class="QDialog" name="PreferenceDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>282</width>
    <height>208</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources/icons.qrc">
    <normaloff>:/icons/setting_tools.png</normaloff>:/icons/setting_tools.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>default language</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::FieldsStayAtSizeHint</enum>
      </property>
      <property name="rowWrapPolicy">
       <enum>QFormLayout::DontWrapRows</enum>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>source language</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="sourceComboBox"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Target language</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="targetComboBox"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>option</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <item row="0" column="0">
       <widget class="QCheckBox" name="autoPastCheckBox">
        <property name="text">
         <string>past automaticaly the mouse selection</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="autoTranslateCheckBox">
        <property name="text">
         <string>Translate directly after past text to translate</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
       <property name="icon">
        <iconset resource="resources/icons.qrc">
         <normaloff>:/icons/cancel.png</normaloff>:/icons/cancel.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="saveButton">
       <property name="text">
        <string>Save</string>
       </property>
       <property name="icon">
        <iconset resource="resources/icons.qrc">
         <normaloff>:/icons/accept.png</normaloff>:/icons/accept.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources/icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>PreferenceDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>291</x>
     <y>367</y>
    </hint>
    <hint type="destinationlabel">
     <x>172</x>
     <y>362</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
